eloquentjs/2_1_looping_a_triangle.js

```diff
@@ -0,0 +1,12 @@
+function DrawTriangle(height){ // Draws a right triangle
+    let tri = '';
+    for(let i =0; i <= height; i++){
+        for(let j =0; j< i; j++){
+            tri+="#";
+        }
+        tri+="\n";
+    }
+    console.log(tri);
+}
+
+DrawTriangle(7);
```

Comment:
I know the exercise didn't say to, but try testing this with a few more values to improve your understanding of the way the function operates. This is something we're going to do while you go through EJS. It's good to always check the limits on functions, so try 0 and 1. You can add a comment or check at the top of the function if you want to disallow negatives, but I don't think the spec makes it important.